### PR TITLE
[INLONG-7480][CVE] Add a specified tomcat-embed-core dependency for fixing the CVE problem

### PR DESCRIPTION
**Pull request by CVEDetect**
### Prepare a Pull Request

- Fixes: #7480

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -215,6 +215,11 @@
                 <artifactId>flume-ng-core</artifactId>
                 <version>${flume.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.tomcat.embed</groupId>
+                <artifactId>tomcat-embed-core</artifactId>
+                <version>8.5.63</version>
```
Comment (dockerzhang) [inline pom.xml:221]: please add the version definition to the root `pom`, it will help keep the same version for other using scenes.

**Comment by dockerzhang** [comment]
@CVEDetect thanks for your contribution, please update by referring to the comment.

**Comment by CVEDetect** [comment]
> @CVEDetect thanks for your contribution, please update by referring to the comment.



> @CVEDetect thanks for your contribution, please update by referring to the comment.

The first commit is not suitable, because there are two tomcat versions 8.5.46 and 9.0.60 in root pom.xml, and the vulnerability exists in inlong-dataproxy, so choose the tomcat version in inlong-dataproxy/pom.xml modified

**Comment by dockerzhang** [comment]
@CVEDetect the `Code Format Check` workflow failed, please run `mvn spotless:apply` to fix it and re-push, thanks.

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**